android/apollo/src/com/andrew/apollo/utils/MusicUtils.java

```diff
@@ -1534,7 +1535,8 @@ public static void refresh() {
      * @return The last album name played by an artist
      */
     public static String getLastAlbumForArtist(final Context context, final String artistName) {
```

Comment:
here I wouldn't touch this method, I'd create another method:

`getLastSongForArtist(...`)`

because the method was doing what was intented, and now you want something else and you're making it do something it's not meant to.